asapdiscovery-docking/asapdiscovery/docking/docking_v2.py

```diff
@@ -156,6 +196,43 @@ def write_docking_files(
             combined_oemol = result.to_posed_oemol()
             save_openeye_pdb(combined_oemol, output_pdb_file)
 
+            results_df = result.make_df_from_docking_results([result])
+            results_df.to_csv(compound_dir / "docking_results.csv")
+
+    def check_results_exist(
```

Comment:
Lets use a save and read of the `DockingResults` JSON to easily regen work we have already done.